src/main/java/data/JsonConverter.java

```diff
@@ -35,12 +35,21 @@ public JsonConverter(String filePathName) {
     basePath = filePathName;
   }
 
+  /**
+   * Checks to see if a vendor's configuration already exists, and updates
+   * it if it does
```

Comment:
Add period.

Reply:
+1 to this comment

Reply:
Also, describe what it does if the config doesn't exist